internal/templates/antiaffinity/internal/params/params.go

```diff
@@ -0,0 +1,5 @@
+package params
+
+// Params represents the params accepted by this template.
+type Params struct {
```

Comment:
Can you add add `MinReplicas` as a parameter to this template, so that the users can specify a minimum number of replicas for which they want to enforce this? We can then specify the value as 2 explicitly in the built-in check.

Reply:
Added..but not sure how to use it.